docs/current_docs/configuration/cloud.mdx

```diff
@@ -11,6 +11,10 @@ import TabItem from "@theme/TabItem";
 Dagger Cloud requires a GitHub or Google account for identity verification. If you don't have one, [register for a free GitHub account](https://github.com/signup) or [create a free Google account](https://accounts.google.com/) before proceeding.
 :::
 
+:::info
+Dagger Cloud requires an engine with version v0.11 or higher.
```

Comment:
```suggestion
Dagger Cloud requires Dagger v0.11 or higher.
```

Reply:
Suggesting this because we don't publish Engine and CLI versions separately, for an end-user there is one product and one version number.

Reply:
Makes sense, thanks!